test/cli.js

```diff
@@ -40,7 +40,9 @@ function execCli(args, opts, cb) {
 
 		child.on('close', (code, signal) => {
 			if (code) {
-				const err = new Error(`test-worker exited with a non-zero exit code: ${code}`);
+				const err = new Error(
+					`test-worker exited with a non-zero exit code: ${code}`,
```

Comment:
Looks like you're using Prettier. Could you exclude unrelated changes from the PR?

Reply:
Got it!